### PR TITLE
add more typings

### PR DESCRIPTION
**Pull request by emilianox**
Hi, I want to add more and better typing to all files, if you like this sample I will continue adding these changes.
All comments and fixes are welcome.
Thanks for all the good work!

### CODE REVIEW

File: src/react/components/connect.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ export class Connect extends React.Component<any> {
     }
   };
   render() {
+    //@ts-ignore
```
Comment (matheusml) [inline src/react/components/connect.tsx]: Why do we need this?

**Comment by matheusml** [comment]
This is great @emilianox 

Please add more, if you'd like.

Btw, before I can merge this, could you please:
- Bump the version to 4.13.2
- Add this change to the CHANGELOG file

File: src/react/components/connect.tsx (hunk 1 above)
Comment (emilianox) [reply src/react/components/connect.tsx]: tsc shows me an error:
Cannot invoke an expression whose type lacks a call signature. Type 'ReactNode' has no compatible call signatures.

**Comment by emilianox** [comment]
added fixes in all the project except preact, stage 1 done, next steps:
add preact typings
replace all any and object types
add types for Actions(requires typescript 3), Connected Component,Middleware etc
move to strict in tsconfig.json

File: src/devtools/devtoolsMiddleware.ts [hunk 2]
```diff
@@ -1,24 +1,71 @@
-let devTools = { instance: null };
+import Store from "../interfaces/Store";
```
Comment (matheusml) [inline src/devtools/devtoolsMiddleware.ts]: What about moving all those Interfaces to a different file?

Maybe create an interface/ folder inside devtools/ folder, one interface = one new file.

What do you think?

Comment (emilianox) [reply src/devtools/devtoolsMiddleware.ts]: done!

**Pull request merged**